tests/components/controlBar-test.js

```diff
@@ -112,7 +112,136 @@ describe('ControlBar', function () {
     TestUtils.Simulate.click(fullscreenButton);
     expect(fullscreenToggled).toBe(true);
   });
+  
+  it('render one stereo button if content vr', function () {
+    var mockController = {
+      state: {
+        isMobile: true,
+        volumeState: {
+          volume: 1
+        },
+        closedCaptionOptions: {},
+        videoQualityOptions: {
+          availableBitrates: null
+        }
+      },
+      videoVrSource: {
+        stereo: false,
+        contentType: "single",
+        startPosition: 0
+      }
+    };
+    
+    var toggleSkinConfig = Utils.clone(skinConfig);
+    toggleSkinConfig.buttons.desktopContent = [{"name":"stereo", "location":"controlBar", "whenDoesNotFit":"keep", "minWidth":35 }];
+    
+    
+    var mockProps = {
+      isLiveStream: false,
+      controller: mockController,
+      skinConfig: toggleSkinConfig,
+      duration: 30,
+      vr: mockController.videoVrSource
+    };
+    
+    var DOM = TestUtils.renderIntoDocument(
+      <ControlBar {...mockProps} controlBarVisible={true}
+                  componentWidth={500}
+                  playerState={CONSTANTS.STATE.PLAYING}
+                  isLiveStream={mockProps.isLiveStream} />
+    );
+    
+    var toggleStereoButton = TestUtils.findRenderedDOMComponentWithClass(DOM, 'oo-vr-stereo-button');
+    expect(typeof toggleStereoButton).toBe('object');
+  });
+  
+  it('not render stereo button if content not vr', function () {
+    var mockController = {
```

Comment:
Same comment as above, we shouldn't re-declare these each time.

Reply:
this is fixed in userstory 1424